src/Components/test/testassets/TestServer/Components.TestServer.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <CompileUsingReferenceAssemblies>false</CompileUsingReferenceAssemblies>
```

Comment:
This one project had an intersection of Components from the shared fx and Components as a package via Blazor. Using the ref assembly to compile here caused version mismatch during compile. This seems to be the only project that's affected by the workaround.